src/pages/cart/Cart.jsx

```diff
@@ -1,125 +1,163 @@
+      price: 960000,
+      number: 2,
+      photo: "https://cdn.imweb.me/thumbnail/20220207/6c05fee561509.jpg",
+      id: 1,
+    },
+    {
+      name: "Vintage Mini Televison",
+      price: 154000,
+      number: 3,
+      photo: "https://cdn.imweb.me/thumbnail/20220207/85ea9b194b0c1.jpg",
+      id: 2,
+    },
+    {
+      name: "Ocean Goblet",
+      price: 45000,
+      number: 1,
+      photo: "https://cdn.imweb.me/thumbnail/20220301/f672ab1fb0f0c.jpg",
+      id: 3,
+    },
+  ]),
 );
-localStorage.setItem(
-  "1",
-  JSON.stringify({
-    name: "Le Corbusier LC2 Chair",
-    price: 960000,
-    number: 2,
-    photo: "https://cdn.imweb.me/thumbnail/20220207/6c05fee561509.jpg",
-  }),
-);
-localStorage.setItem(
-  "2",
-  JSON.stringify({
-    name: "Vintage Mini Televison",
-    price: 154000,
-    number: 3,
-    photo: "https://cdn.imweb.me/thumbnail/20220207/85ea9b194b0c1.jpg",
-  }),
-);
-const bag = (
-  // () => {
-  //   const [length, setlength] = useState(localStorage.length);
-  //   const Decrease = () => {
-  //     setlength((prevNumber) => prevNumber - 1);
-  //   };
-  //   return (
-  <div>
-    장바구니 <span>{localStorage.length}</span>
-  </div>
-);
-//   );
-// };
 
-const table1Title = (
-  <div>
+const list = JSON.parse(localStorage.getItem("item"));
+
+const FreeDelivery = () => {
+  return (
     <div>
-      {localStorage ? <input type="checkbox" /> : <span> </span>}
-      <div>상품 정보</div>
+      <div>
+        무료
+        <span>
+          <button type="button">?</button>
+        </span>
+      </div>
+      <div>택배</div>
     </div>
-    <div>수량</div>
-    <div>주문금액</div>
-    <div>배송 정보</div>
-  </div>
-);
-
-const lo = [];
-const arr = Array(localStorage.length)
-  .fill()
-  .map((arr, i) => {
-    return i;
-  });
-arr.map((x) => lo.push(JSON.parse(localStorage[x])));
-console.log(lo);
+  );
+};
 
-const remove = (event) => {
-  if (event.target.closest(".del")) {
-    localStorage.removeItem(event.target.closest(".item").classList[1]);
-    event.target.closest(".item").remove();
-    lo.splice(`${event.target.closest(".item").classList[1]}`, 1);
-  }
+const Delivery = () => {
+  return (
+    <div>
+      <div>
+        2500원
+        <span>
+          <button type="button">?</button>
+        </span>
+      </div>
+      <div>택배</div>
+    </div>
+  );
 };
 
 const Cart = () => {
+  const [items, setItems] = useState(list);
+
+  const deleteHandler = (name) => {
+    setItems(items.filter((item) => item.name !== name));
+  };
+
+  useEffect(() => {
+    localStorage.setItem("item", JSON.stringify(items));
+  }, [items]);
+
+  const sumItems = items.reduce(
+    (acc, item) => acc + item.price * item.number,
+    0,
+  );
+
+  const [checkItems, setCheckItems] = useState([]);
+  const handleSingleCheck = (checked, id) => {
+    if (checked) {
+      setCheckItems((prevCheckItems) => [...prevCheckItems, id]);
+    } else {
+      setCheckItems(checkItems.filter((el) => el !== id));
```

Comment:
```suggestion
      setCheckItems(checkItems.filter((checkItem) => checkItem !== id));
```

el 대신 checkItem 으로 파라미터를 바꾼다면 좀 더 가독성이 올라가지 않을까요?